src/components/nodes/sequenceFlow/sequenceFlow.vue

```diff
@@ -26,6 +27,20 @@ export default {
       return get(this.target, 'component.node.type');
     },
   },
+  watch: {
+    'node.definition.conditionExpression.body'(conditionExpression) {
+      this.shape.labels([{
```

Comment:
The label can be updated with the following function, this way it's position won't be reset when updating:
```js
this.shape.label(0, {
  attrs: {
    text: {
      text: conditionExpression,
    },
  },
});
```